README.md

```diff
@@ -82,20 +82,20 @@ paymentForm.addEventListener("submit", payWithPaystack, false);
 function payWithPaystack(e) {
   e.preventDefault();
 
-  let handler = PaystackPop.setup({
-    key: 'pk_test_xxxxxxxxxx',
-    email: document.getElementById("email-address").value,
-    amount: document.getElementById("amount").value * 100,
-    ref: paystackTxRef // Reference returned from plugin
-    onClose: function(){
-      alert('Window closed.');
-    },
-    callback: function(response){
-      // Call Medusa checkout complete here
-    }
-  });
-
-  handler.openIframe();
+  const paystack = new PaystackPop();
+
+  paystack.newTransaction({
+	key: 'pk_test_xxxxxxxxxx', // Your Paystack public key
+	email: document.getElementById("email-address").value,
+	amount: document.getElementById("amount").value * 100,
+  ref: paystackTxRef, // Reference gotten from plugin
+  onSuccess(){
+    // Call Medusa checkout complete here
+  }
+  onCancel(){
+    alert("Window closed.")
+  }
+});
```

Comment:
Could you update the indent here, they don't seem aligned

Reply:
Will fix that